### PR TITLE
Return null instead of false when unable to load model.

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

**Review by dereuromark — APPROVED** [review]

File: src/Controller/Controller.php [hunk 1]
```diff
@@ -304,7 +304,7 @@ public function __get(string $name)
             E_USER_NOTICE
         );
 
-        return false;
+        return null;
```
Comment (ravage84) [inline src/Controller/Controller.php]: Can't we throw a new specific exception in that case, instead of triggering an error?

Comment (dereuromark) [reply src/Controller/Controller.php]: As long as we dont have a has() method for it, I dont think we can.

Comment (ravage84) [reply src/Controller/Controller.php]: Why do you think we would need a ``has()`` method before we can throw an exception in this failure case?

Comment (dereuromark) [reply src/Controller/Controller.php]: Fair enough. Not sure about all use cases, but I guess being a bit stricter doesn't hurt after all.

Comment (ravage84) [reply src/Controller/Controller.php]: Yeah, I understand where you wanted to point at, but I tried to think of a scenario where it was better to not throw an exception but just triggering an error... Didn't find one. But when a model couldn't be loaded, it's likely that sh*t hits the fan, sooner or later ...

Also, this is for 4.x where we can easily introduce something like this.

Comment (ADmad) [reply src/Controller/Controller.php]: There's no need for an exception. Showing notice when trying to use non-existent property is default PHP behavior and same has been done. `loadModel()` already throws an exception it's unable to load a model.

Comment (garas) [reply src/Controller/Controller.php]: I would even remove `__get()` and require developer to explicitly load controller’s default model.

Comment (cleptric) [reply src/Controller/Controller.php]: I don’t like this idea. Explicity moves us further away from our “rapid development” approach without a clear benefit.

**Review by saeideng — APPROVED** [review]

**Pull request merged**